RevenueCat/Plugins/Android/com/revenuecat/purchasesunity.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I moved the files to the correct folder `com/revenuecat/purchasesunity` so their package matches the folder